Tensile/SolutionStructs.py

```diff
@@ -942,14 +942,16 @@ def checkIfSupportedGEMMType(self):
     if inType.isHalf():
       # TODO- Test and migrate ([H/H/H]+HPA) to ([H/H/S]+HPA)
       # Note that we need to do a little change in rocBLAS and logic yaml
-      if not (outType.isHalf() and (computeType.isHalf() or computeType.isSingle())):
-        printExit("DataType=H only allows DestDataType=H and ComputeDataType=H/S")
+      if not ((outType.isHalf() or outType.isSingle()) and \
+              (computeType.isHalf() or computeType.isSingle())):
+        printExit("DataType=H only allows DestDataType=H/S and ComputeDataType=H/S")
       # if not (outType.isHalf() and computeType.isHalf()):
       #   printExit("DataType=H only allows DestDataType=H and ComputeDataType=H")
 
     if inType.isBFloat16():
-      if not (outType.isBFloat16() and computeType.isSingle()):
-        printExit("DataType=B only allows DestDataType=B and ComputeDataType=S")
+      if not ((outType.isBFloat16() or outType.isSingle()) and  \
+              computeType.isSingle()) :
+        printExit("DataType=B only allows DestDataType=B/S  and ComputeDataType=S")
 
```

Comment:
I am wondering how can I benchmark the new type and the validation?  Since I don't see any changes or new typed added in tensile client code.

Reply:
Sorry my bad, I figure that the HSS and BSS APIs are already implemented before. But not HSH, so maybe we have to reject HSH in checkIfSupportedGEMMType() in SolutionStruct.py

Reply:
#1144 did this already in my prep for this PR. Here I also added the configs so that we might actually test BSS and HSS

Reply:
- Fixed.